db/db_compaction_test.cc

```diff
@@ -3673,10 +3672,8 @@ TEST_F(DBCompactionTest, LevelTtlCascadingCompactions) {
             }
           });
 
-      env_->time_elapse_only_sleep_ = false;
       options.env = env_;
 
-      env_->addon_time_.store(0);
```

Comment:
I forgot to mention that in the description. I intentionally dropped this in many cases, where I did not believe it would be needed. How about I put a comment like this in those places:

    // NOTE: Presumed unnecessary and removed: resetting mock time in env